mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -707,6 +708,30 @@ public void testMultiTableInsert() throws IOException {
     HiveIcebergTestUtils.validateData(target2, target2Records, 1);
   }
 
+  @Test
+  public void testStructMapWithNull() throws IOException {
+    Assume.assumeTrue("Failed on vectorized parquet for a bug in parquet vectorization",
+        !("PARQUET".equals(fileFormat.name()) && isVectorized));
```

Comment:
Nit:  Instead of negating on `assumeTrue`, why not just use `assumeFalse`? Either that or distributing the `not` operator, but I think the following is much easier to read.

```java
    Assume.assumeFalse("...", "PARQUET".equals(fileFormat.name()) && isVectorized);
```

Reply:
Also, can you please update the `Assume` statement to provide more details on what the bug is and why we can't run this test under these conditions?

The assume statement should provide enough details and context to know how to follow up if possible, or at least what the bug is. For example, some of the other `Asssume` statements in this file say `Tez is not implemented yet` which is straightforward. But I don't know what the vectorized parquet bug is when seeing this and I'm left with more questions. Is it something that's not yet implemented, or maybe there's an issue that can be referenced from the parquet project? A short high level summary of what would happen and possibly a link to an existing parquet-ticket or something would be really appreciated.  Maybe this is something that can be fixed?

In general, the assume statements and error messages should be written in a way that we have enough context to understand why we're skipping the test and where to follow up if need be.

Reply:
Hi @kbendick, sorry for the delay,
I create an issue for this https://github.com/apache/iceberg/issues/4403 and add the info inside the test.